### PR TITLE
Refresh Explanations/Security

### LINKED ISSUE #344: Review and update Exp/Security

### PR DESCRIPTION
**Pull request by jhkrug**
I have a query in the TODO:. Thanks,

Will close #344 when merged.

### CODE REVIEW

File: docs/explanations/security.md [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
+title: "Epinio security"
+description: Epinio security, concepts and context
+keywords: [epinio, kubernetes, security, application development]
 ---
 
-## A Quick Note On Security
+<!--TODO:
+Should we be saying anything about source code level security, automated
+with Dependabot, Snyk, whatever?
+-->
```
Comment (andreas-kupries) [inline docs/explanations/security.md:11]: Oh. ... On one hand I see this as details the user has no need to know of.
On the other, doing so may be seen as an indicator of us taking the topic serious.
@olblak I suspect that this is more your level of things, i.e. beyond technical.

File: docs/explanations/security.md [hunk 2]
```diff
@@ -1,32 +1,42 @@
...
-Use the `epinio login [URL]` command after installation to save the necessary credentials
-(user, password) and certificates. The information is stored in Epinio's settings,
-for pickup by other Epinio commands.
+Use the `epinio login [URL]` command after installation to save the necessary credentials (user, password) and certificates.
+Epinio stores credentials in its settings for use by other Epinio commands requiring them.
 
-For a trial deployment the certificate securing the API will be generated by the
-underlying cluster, and self-signed, and its CA certificate is stored in the
-settings to allow verification.
+For a trial deployment the underlying cluster generates certificate securing the API.
```
Comment (andreas-kupries) [inline docs/explanations/security.md:18]:
```suggestion
For a trial deployment the underlying cluster generates a certificate securing the API.
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]

**Review by andreas-kupries — DISMISSED** [review]

**Comment by enrichman** [comment]
I would add a few lines regarding `cosign`. We are using it to sign the Epinio images.

We have this instructions about how to check the verification but nothing more, so it makes sense: https://deploy-preview-367--epinio-docs-staging.netlify.app/installation/install_epinio_cli#verify-downloaded-files

Maybe we can see what Kubewarden does, they have the same.

File: docs/explanations/security.md (hunk 1 above)
Comment (jhkrug) [reply docs/explanations/security.md:11]: OK, thanks, will merge for the moment.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**